plugin.py

```diff
@@ -17,6 +21,25 @@ class LspTailwindcssPlugin(NpmClientHandler):
         server_directory, 'extension', 'dist', 'server', 'index.js'
     )
 
+    @classmethod
+    def is_allowed_to_start(
+        cls,
+        _window: sublime.Window,
+        _initiating_view: Optional[sublime.View] = None,
+        workspace_folders: Optional[List[WorkspaceFolder]] = None,
+        _configuration: Optional[ClientConfig] = None
+    ) -> Optional[str]:
+        if not workspace_folders:
+            return "{} - requires a folder to start.".format(cls.package_name)
+        path = workspace_folders[0].path
+        tailwind_config_file_path = os.path.join(path, 'tailwind.config.js')
+        if not os.path.exists(tailwind_config_file_path):
+            return "{} - no tailwind.config.js present in {}".format(cls.package_name, path)
+        is_tailwind_dependecy_installed = os.path.join(path, 'node_modules', 'tailwindcss')
+        if not os.path.exists(is_tailwind_dependecy_installed):
+            return "{} - No tailwindcss package found in node_modules for path {}. Make sure to install the tailwindcss npm package.".format(cls.package_name, path)
```

Comment:
The `{} -` prefixes with the package name are not necessary. Other than that LGTM

Reply:
aafd2ea